lib/easel_provider.dart

```diff
@@ -102,12 +103,16 @@ class EaselProvider extends ChangeNotifier {
 
   Denom get selectedDenom => _selectedDenom;
 
-  final artistNameController = TextEditingController();
-  final artNameController = TextEditingController();
-  final descriptionController = TextEditingController();
-  final noOfEditionController = TextEditingController();
-  final priceController = TextEditingController();
-  final royaltyController = TextEditingController();
+  String get recipeId => _recipeId;
+
+  String? get cookbookId => _cookbookId;
+
+  TextEditingController? artistNameController = TextEditingController();
```

Comment:
Why all these are now nullable?

Reply:
Because, as we are saving NFT in floor when a user uploads an image so we have to send these values in NFT model and we are taking these values in the next screen.